packages/integration-test/test/native-token.test.ts

```diff
@@ -5,42 +5,34 @@ import { AdvancedLogic } from '@requestnetwork/advanced-logic';
 import { CurrencyManager } from '@requestnetwork/currency';
 
 const advancedLogic = new AdvancedLogic();
-const currency = {
-  network: 'aurora-testnet',
-  type: RequestLogicTypes.CURRENCY.ETH,
-  value: 'NEAR',
-};
+
 const createCreationActionParams: PnReferenceBased.ICreationParameters = {
   paymentAddress: 'payment.testnet',
   salt: 'a1a2a3a4a5a6a7a8',
   paymentNetworkName: 'aurora-testnet',
 };
 
 describe('PaymentNetworkFactory and createExtensionsDataForCreation', () => {
+  const paymentNetworkFactory = new PaymentNetworkFactory(
+    advancedLogic,
+    CurrencyManager.getDefault(),
+  );
   it('PaymentNetworkFactory can createPaymentNetwork (mainnet)', async () => {
-    const paymentNetwork = PaymentNetworkFactory.createPaymentNetwork({
-      advancedLogic,
-      currency: { ...currency, network: 'aurora-testnet' },
-      paymentNetworkCreationParameters: {
-        id: PaymentTypes.PAYMENT_NETWORK_ID.NATIVE_TOKEN,
-        parameters: createCreationActionParams,
-      },
-      currencyManager: CurrencyManager.getDefault(),
-    });
+    const paymentNetwork = paymentNetworkFactory.createPaymentNetwork(
+      PaymentTypes.PAYMENT_NETWORK_ID.NATIVE_TOKEN,
+      RequestLogicTypes.CURRENCY.ETH,
+      'aurora-testnet',
+    );
     const action = await paymentNetwork.createExtensionsDataForCreation(createCreationActionParams);
     expect(action.parameters.paymentAddress).toEqual('payment.testnet');
     expect(action.parameters.paymentNetworkName).toEqual('aurora-testnet');
   });
   it('throws without a payment network name', async () => {
```

Comment:
for me it seems the same execution than the previous test so I don't see where the paymentNetworkName is removed from createCreationActionParams as it was before.